INSTALL.md

```diff
@@ -20,78 +20,87 @@ The system requirements depend on how you install UDOIT.  If you use Docker, the
+	- `APP_LMS`:
+	   * `canvas` for Canvas LMS.
+	   * `d2l` for D2l Brightspace LMS.
+	- `JWK_BASE_URL`: If you are self-hosting Canvas, you may set it to the URL of your instance of Canvas. (Example: `JWK_BASE_URL="https://canvas.dev.myschool.edu"`)
+	- `DEFAULT_LANG`: (optional) the default language of UDOIT. This is English by default.
+		-  `en` for English
+		-  `es` for Spanish.
+
+## Installation
+
+### Option 1. Docker
+We provide a fast and simple way of setting up a local UDOIT instance through Docker.
+
+1. Install [Docker Desktop](https://docs.docker.com/get-docker/). This will install Docker and Docker Compose on your system. 
+	> Alternatively, you may install Docker and [Docker Compose](https://docs.docker.com/compose/install/) individually.
+
+3. Build the Containers
+If you prefer to build the containers yourself, or you are actively developing UDOIT and need to rebuild the containers to test your changes, run the following command from within the UDOIT directory:
+```
     docker compose -f docker-compose.nginx.yml up
+```
 
-### 3. Wait
-Wait for all of the containers to finish initializing.  This can take over 15 minutes.  You will know you are ready to proceed with the next step when you haven't seen any output in your terminal for a few minutes.
-
-### 4. Set up the Database
-The first time you start the containers, you will need to set up the database to handle all the information UDOIT generates as it runs.  Run the following command:
+4. Wait for the containers to finish initializing. This may take over 15 minutes. If you see no output in your terminal for few minutes, proceed to the next step.
 
+5. Once the containers are intialized, run the following command:
+```
     docker compose -f docker-compose.nginx.yml run php php bin/console doctrine:migrations:migrate
+```
+This applies migrations necessary to set up the database to store all UDOIT data.
 
-> You will also need to run that command whenever you update to a new version of UDOIT.
+Running this will give the following warning:
+ > WARNING! You are about to execute a migration in database "udoit3" that could result in schema changes and data loss. Are you sure you wish to continue? (yes/no) [yes]:
 
-### 5. Next steps
-Skip to [Testing your Setup](#testing-your-setup) to continue.
+Don't worry! Simply type `yes` and proceed.
 
-### 6. Stopping the Containers
-If you ever want to stop the containers, you can do so with the following command:
-
-    docker compose -f docker-compose.nginx.yml down
+> You will need to run this command whenever you update to a new version of UDOIT.
 
+5. UDOIT should be installed and running in your Docker cotnainer. 
 
+6. To stop UDOIT, run the following command:
```

Comment:
```suggestion
6. To stop the UDOIT containers, run the following command:
```
Taking out the section headings, I'd prefer to be more verbose in the instructions, so the user has a better idea of what they'll be executing. This'll also help us when we're troubleshooting issues for users that come to us for help.